syntax = "proto3";

message PrintDatastoreReq {
  int32 clusterId = 1;
  int32 serverId = 2;
}